src/passes/LegalizeJSInterface.cpp

```diff
@@ -29,6 +29,11 @@
 // across modules, we still want to legalize dynCalls so JS can call into the
 // tables even to a signature that is not legal.
 //
+// Another variation also "prunes" imports and exports that we cannot yet
+// legalize, like exports and imports with SIMD or multivalue. Until we write
```

Comment:
Why would we need to legalize or prune out multivalue? We should only be testing on JS engines that fully support it, right?